src/components/WalletSection.tsx

```diff
@@ -1,29 +1,22 @@
-import PropTypes from 'prop-types';
-import React from 'react';
+import React, {ReactNode} from 'react';
 import useThemeStyles from '@styles/useThemeStyles';
 import Section from './Section';
 
-const propTypes = {
+type WalletSectionProps = {
     /** Contents to display inside the section */
-    children: PropTypes.node,
+    children?: ReactNode;
```

Comment:
I think you can use [ChildrenProps](https://github.com/Expensify/App/blob/main/src/types/utils/ChildrenProps.ts) type instead